js/xt.js

```diff
@@ -0,0 +1,1072 @@
+        /**
+         * @method
+         * @name xt#fetchMarkets
+         * @description retrieves data on all markets for xt
+         * @param {object} params extra parameters specific to the exchange api endpoint
+         * @returns {[object]} an array of objects representing market data
+         */
+        // let type = undefined;
+        // [ type, params ] = this.handleMarketTypeAndParams ('fetchMarkets', undefined, params);
+        const query = this.omit (params, 'type');
+        // if (type === 'spot') {
+        //     // spot and swap ids are equal
+        //     // so they can't be loaded together
+        //     const spotMarkets = await this.fetchSpotMarkets (query);
+        //     return spotMarkets;
+        // }
+        return this.fetchSwapAndFutureMarkets (query);
+    }
+
+    async fetchSpotMarkets (params = {}) {
+        const response = await this.spotPublicGetDataApiV1GetMarketConfig (params);
+        //
+        //    {
+        //      "raca_xt":
+        //      {
+        //      "minAmount": 1,
+        //      "pricePoint": 7,
+        //      "coinPoint": 0,
+        //      "maker": 0.002,
+        //      "minMoney": 0,
+        //      "taker": 0.002
+        //      }
+        //    }
+        //
+        const result = [];
+        const ids = Object.keys (response);
+        for (let i = 0; i < ids.length; i++) {
+            const id = ids[i];
+            const market = this.safeValue (response, id);
+            const baseId = id.split ('_')[0];
+            const quoteId = id.split ('_')[1];
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const symbol = base + '/' + quote;
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'settle': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'settleId': quoteId,
+                'type': 'spot',
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': true,
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'taker': this.safeNumber (market, 'taker'),
+                'maker': this.safeNumber (market, 'maker'),
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'feeCurrency': quote,
+                // TODO: Precision
+                'precision': {
+                    'amount': undefined, // lot
+                    'price': undefined, // step
+                },
+                'limits': {
+                    'amount': {
```

Comment:
I'd say `minAmount` and `minMoney` could be used here inside `limits` 
![image](https://user-images.githubusercontent.com/43336371/192585088-b53006e1-ad2b-4a92-b970-1bdbc3329562.png)

Reply:
To what value does `minMoney` related to? Minimum cost?

Reply:
Yeah I'd say so (although it's not very clear)